httpie/output/ui/palette.py

```diff
@@ -0,0 +1,138 @@
+# Copy the brand palette
+
+COLOR_PALETTE = {
```

Comment:
`PIE_COLOR_PALETTE`

Reply:
Do we need this prefix? I assumed these were going to be the default colors we use after this PR (e.g on progress bars, on regular colored outputs etc.)

Reply:
Good question. I thought we’d always re-use the effective `--style` even for the UI, because if I use the terminal scheme, for example, then I’d like to see those colors everywhere. In that case, giving it the `PIE_` prefix would make sense.

Reply:
I especially put this into a separate package, `httpie.output.ui`, since I plan to (at least feel like) this is going to be the place where we would manage our rich UI (progress bars, themes, highlighting, console, etc.), and those probably will use this theme. If you feel strong about it, I can add the `PIE_` prefix though but personally I'd find this very compelling (since the fully qualified name of this is `httpie.output.ui.COLOR_PLAETTE`, so it is not something arbitrary).

Reply:
OK. Although we should discuss this:

> I thought we’d always re-use the effective `--style` even for the UI, because if I use the terminal scheme, for example, then I’d like to see those colors everywhere.